docs/content/testing/testing-guidelines.md

```diff
@@ -0,0 +1,265 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing Guidelines
+
+The tests should:
+
+- Set up some initial state
+- Execute a method that acts upon this state and results in a new state
+- Make assertions in the new state against the expected changes
+- Clean up
+
+NUnit allows this to be done through the use of Fixtures and special method annotations.
+
+## Module Directory Structure
+
+The Unity GDK project's assets are divided into modules, such as `Core`, `Physics`, `PlayerLifecycle`.
+
+Each module is in its own directory within `workers\unity\Assets\Gdk`.
+
+Each module has its own tests that cover the functionality of that module.
+
+Unity Test Runner also allows separate playmode and editmode tests. For more information on this division, please refer to the [Unity Test Runner manual page](https://docs.unity3d.com/Manual/testing-editortestsrunner.html).
+
+Each module folder will contain the tests about that module, and within separate subdirectories if a module needs both editmode and playmode testing - this is a requirement from Unity:
+
+> The Editmode and Playmode folders need to be separate as the tests need to be in different assemblies, and currently Unity `.asmdef` files affect all files within the same directory.
+
+The folder structure for a module should look like this:
+- `<ModuleName>` (e.g. `Core`)/
```

Comment:
I find the formatting in this list a bit hard to read

Reply:
Removing this list and writing a sentence instead